### PR TITLE
Backup the local workflows used to load no2 and nightlights

### PR DESCRIPTION
**Pull request by anayeaye**
## What
This PR adds the local workflows used to load no2-monthly and nightlights-hd-monthly datasets into PgStac for the record and for reproducibility if needed.

### CODE REVIEW

File: dataset-workflows/nightlights-hd-monthly/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Python code to ingests the full covid-19 Black Marble High Definition Nightlights dataset into a target PgSTAC database using [pypgstac](https://github.com/stac-utils/pgstac#pypgstac) from a laptop.
+1. The collection json object is flattened to an ndjson array and inserted into pgstac.
+2. A boto3 search is used to identify all COG datasets matching a bucket key pattern.
+3. STAC records are created with raster and projection information for each matching COG using [rio-stac](http://devseed.com/rio-stac/api/rio_stac/stac/#create_stac_item).
+4. An ndarray of all new item records is ingested to pgstac using pypgstac.
+5. Finally, the update_default_summaries dashboard extension function in the pgstac database is executed to update the collection record with dynamic item datetime and default COG raster statistic summaries.
+
+
+## Requirements
+```
```
Comment (abarciauskas-bgse) [inline dataset-workflows/nightlights-hd-monthly/README.md:16]: Should we put this in a requirements.txt file? Here and for the no2 monthly instructions

File: dataset-workflows/nightlights-hd-monthly/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+boto3
+pypgstac
+psycopg[binary]
+pystac
+rio_stac
+tqdm
+```
+
+## Set-up
+Parameters are hard coded in these one-off load routines; here are the parameters that need to be set.
```
Comment (abarciauskas-bgse) [inline dataset-workflows/nightlights-hd-monthly/README.md:26]:
```suggestion
Parameters are hard-coded in these one-off load routines. Here are the parameters that need to be set:
```

File: dataset-workflows/nightlights-hd-monthly/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Confirm that the collection json defined in the file is correct
+   - `item_assets`, `dashboard:is_periodic`, and `dashboad:time_density` are required properties to generate collection statistics.
+2. Confirm that the S3 bucket and prefix are correct
+3. Set AWS secret key name (or ARN) to lookup database credentials in target environment
+4. [Optional] Set AWS profile name if not using default profile
+5. Run once with `dry_run=True` and verify the ndjson outputs for accuracy. When dry_run is false, the records will be published to the target database and the local files will be removed.
```
Comment (abarciauskas-bgse) [inline dataset-workflows/nightlights-hd-monthly/README.md:33]:
```suggestion
5. Run once with `dry_run=True` and verify the ndjson outputs for accuracy. When `dry_run=False`, the records will be published to the target database and the local files will be removed.
```

**Comment by abarciauskas-bgse** [comment]
@anayeaye I proposed the change to consolidate the functions - would you prefer to make the changes now, merge as-is and open an issue to refactor or close and open an issue to revisit?

**Comment by anayeaye** [comment]
There are now automated pipelines for these datasets, closing this PR.

**Comment by anayeaye** [comment]
@abarciauskas-bgse I just re-read your comment--do you think there is value to polishing these one off tools/consolidating functions? I closed assuming that the step functions tasks were able to cover all ingest needs at this point but I'd be happy to move some/all of this to a backlog issue.

**Comment by abarciauskas-bgse** [comment]
@anayeaye I think there would be value but given there is some refactoring going on and more pressing issues, I wouldn't spend time on it right now. We can return to this PR if necessary.

**Pull request closed without merging**